modules/proc_use-api.adoc

```diff
@@ -104,6 +101,82 @@ Use the following procedure to create an OAuth access token.
 This is the only opportunity to copy and save the access token. It cannot be reobtained after leaving this page. 
 ====
 
+[id="reassigning-oauth-access-token"]
+== Reassigning an OAuth access token
+
+Organization administrators can take control of OAuth API tokens created by other users and re-assign those tokens to a different user. This helps audit logs properly reflect which member of an organization used, or is using, a token after reassignment.
```

Comment:
This isn't entirely accurate. In the implementation an organization administrator can permit a different user to create a token for themselves, even if they don't have organization administrator privileges. Previously only org administrators could create tokens, so they would create tokens on behalf of other users but in the logs it would show up as the org administrator doing the action, when really the token was being used by someone else.

In this feature the org administrator can basically say `user X can create a token for themselves with permissions Y`. Then the user has permissions to create a token with the assigned permissions for themselves. So a better description would be:
`Organization administrators can assign OAuth API tokens to be created by other user's with specific permissions. This allows the audit logs to be reflected accurately when the token is used by a user that has no organization admin permissions to create a OAuth API token.

Reply:
Fixed.